key.core/src/test/java/de/uka/ilkd/key/proof/runallproofs/proofcollection/ForkedTestFileRunner.java

```diff
@@ -174,7 +174,7 @@ public static void main(String[] args) throws IOException {
                     testResults.add(testFile.runKey());
                 } catch (Exception e) {
                     error = true;
-                    e.printStackTrace();
+                    LOGGER.warn("Run failed", e);
```

Comment:
A general question: Will ".warn" be usually printed to the console by default?

Reply:
Our log level is >= Info for all projects I know of, so yes. Any printing using `System.err` (which includes `e.printStackTrace`) is not included in any log files generated by KeY so they are lost after closing the console, so this code definitely should go. The only question is whether all log levels are configured correctly.